src/main/java/Group15/HomeScreenView.java

```diff
@@ -69,9 +70,15 @@ private static Pane createButtonPane() {
             newButton.setPrefSize(200, 50);
             newButton.setOnAction(_ -> {
                 switch (button) {
-                    case "New Workout" -> System.out.println("New Workout pressed");
-                    case "My Workouts" -> System.out.println("My Workouts pressed");
-                    case "Exit" -> System.exit(0);
+                    case "New Workout":
+                        Scene createWorkoutScene = MuscleSelectionView.createMuscleSelectorScene();
```

Comment:
You should also use the viewController here

Reply:
agreed